crc32_braid_p.h

```diff
@@ -3,60 +3,36 @@
 
 #include "zendian.h"
 
-/* Define N */
-#ifdef Z_TESTN
-#  define N Z_TESTN
-#else
-#  define N 5
-#endif
-#if N < 1 || N > 6
-#  error N must be in 1..6
-#endif
+/* Define N, valid range is 1..6 */
```

Comment:
"Define BRAID_N" to be consistent with line below...

Reply:
Fixed.